jbpm-designer-backend/src/main/java/org/jbpm/designer/web/profile/impl/EpnProfileImpl.java

```diff
@@ -1,361 +0,0 @@
-/*
- * Copyright 2015 Red Hat, Inc. and/or its affiliates.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * 
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
-*/
-
-package org.jbpm.designer.web.profile.impl;
-
-import java.io.ByteArrayOutputStream;
-import java.io.FileInputStream;
-import java.io.FileNotFoundException;
-import java.io.IOException;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.LinkedHashMap;
-import java.util.Map;
-
-import javax.enterprise.context.ApplicationScoped;
-import javax.servlet.ServletContext;
-import javax.xml.stream.XMLInputFactory;
-import javax.xml.stream.XMLStreamException;
-import javax.xml.stream.XMLStreamReader;
-
-import org.codehaus.jackson.JsonParseException;
-import org.eclipse.bpmn2.Definitions;
-import org.eclipse.emf.ecore.resource.Resource;
-import org.jbpm.designer.bpmn2.impl.Bpmn2JsonUnmarshaller;
-import org.jbpm.designer.bpmn2.resource.JBPMBpmn2ResourceImpl;
-import org.jbpm.designer.epn.impl.EpnJsonMarshaller;
-import org.jbpm.designer.epn.impl.EpnJsonUnmarshaller;
-import org.jbpm.designer.repository.Repository;
-import org.jbpm.designer.util.ConfigurationProvider;
-import org.jbpm.designer.web.plugin.IDiagramPlugin;
-import org.jbpm.designer.web.plugin.impl.PluginServiceImpl;
-import org.jbpm.designer.web.profile.IDiagramProfile;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-
-/**
- * The implementation of the epn profile for Process Designer.
- * @author Tihomir Surdilovic
- */
-@ApplicationScoped
-public class EpnProfileImpl implements IDiagramProfile {
```

Comment:
Could you please remove also `src/main/resources/org/jbpm/designer/public/profiles/epn.xml`?